Dockerfile

```diff
@@ -34,8 +35,10 @@ RUN cp ./ocm-backplane /out
 RUN chmod -R +x /out
 
 ### Build dependencies
-FROM brew.registry.redhat.io/rh-osbs/openshift-golang-builder:rhel_9_golang_1.23 as dep-builder
+FROM brew.registry.redhat.io/rh-osbs/openshift-golang-builder:rhel_9_golang_1.24 as dep-builder
 
+# Ensure we can use Go 1.24.6
+ENV GOTOOLCHAIN=go1.25.3+auto
```

Comment:
Is go 1.25.3 intentional? I see this PR is about updating to 1.24.6

Reply:
Thanks for pointing this out. I intended to keep it on the latest to give us less dep overheads in future as this was 1.24 compatible too. However, I have added 1.24 specific one.